### ISSUE TITLE
Marking unsubscribed computeds  as not fresh

### ISSUE THREAD

**Issue by mrjjwright** [luwes/sinuous#164]
This is almost certainly not true but I have worked a bug all day where a computed stops firing because it is unsubscribed when fresh in an update cycle and so when it's accessed again by parent computations, later in the `runObservers.forEach`, it's update function is not run since it's fresh, and then it falls out of the observable's observers Set.    Adding this line to mark all computations that are unsubscribed as not fresh, probably a horrible idea,  fixes it .   Is there a common reason this happens, I can't figure out what in my code would cause it, but I am sure I am missing something.     Sorry I can't provide a reproducing example outside the sequences of UI actions in my code, I will keep trying.

![image](https://user-images.githubusercontent.com/4343/114288851-f638f380-9a2f-11eb-9b02-ffd35ebc9777.png)

**Comment by luwes** [comment]
thanks for the feedback @mrjjwright!
kinda related to https://github.com/luwes/sinuous/issues/162

maybe computeds need some changes with tracking, they kinda piggyback off the observables it contains 
https://github.com/luwes/sinuous/blob/ece2c65c19939ec5742f999cfba6c83295c58118/packages/sinuous/observable/src/observable.js#L174-L174

**Comment by mrjjwright** [comment]
I was working with a series of nested computations with conditional logic, so that the line you mentioned above would have a different set of observables each time.  I was able to fix this by making sure I accessed all observables in all the computeds all the time instead of hidden underneath conditionals.  I worked this morning on trying to create a test case that reproduced what I was seeing but I couldn't.  I think the implication is that nested computeds with conditionals might in some cases stop reacting unless all the dependent observables are hit each time they update.   Again marking children observers as not `_fresh` in `_unsubscribe` does fix that problem for me so that I don't have to do that.  What happens is that in certain cases a child observer will run it's update earlier in the `runObservers` loop  then it's parents, become fresh, and then it's parents will still unsubscribe them.   Since they are fresh, they will fall out.  I am closing this because this is obviously a bit vague.
